testng.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
+<suite name="Single">
+    <test name="BStackDemo 1">
```

Comment:
Keep the Default parallel browser to Chrome

Reply:
it is, since it will take this json-part:
```

 "single": {
      "common_caps": {
        "project": "BrowserStack",
        "build": "Automate Demo",
        "name": "Single Test: "
      },
      "env_caps": [
        {
          "os": "Windows",
          "os_version": "10",
          "browser": "Chrome"
        }
      ]
    },
```